### PR TITLE
make some of our snowflake tests run in parallel

### PR DESCRIPTION
**Pull request by iskakaushik**

### CODE REVIEW

**Comment by serprex** [comment]
Did you test that failures still fail? According to https://github.com/stretchr/testify/issues/187 testify/suite silently passes parallel tests

File: flow/e2e/snowflake/peer_flow_sf_test.go [hunk 1]
```diff
@@ -5,40 +5,76 @@ import (
...
 	connector *connsnowflake.SnowflakeConnector
 }
 
 func TestPeerFlowE2ETestSuiteSF(t *testing.T) {
-	suite.Run(t, new(PeerFlowE2ETestSuiteSF))
+	s := &PeerFlowE2ETestSuiteSF{}
+	s.SetT(t)
+	s.SetupSuite()
+
+	tests := []struct {
```
Comment (serprex) [inline flow/e2e/snowflake/peer_flow_sf_test.go:39]: #629 added code to `test_utils.go` to generate this kind of table automatically with `reflect`

**Comment by iskakaushik** [comment]
yeah, calling suite.Run seems like the cause of the issue

**Comment by iskakaushik** [comment]
there seems to be one sqlserver test failing, retring to see if its a flake.

**Review by serprex — APPROVED** [review]

**Pull request merged**